google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BigtableCloudMonitoringExporter.java

```diff
@@ -79,11 +78,11 @@ public final class BigtableCloudMonitoringExporter implements MetricExporter {
       Logger.getLogger(BigtableCloudMonitoringExporter.class.getName());
 
   // This system property can be used to override the monitoring endpoint
-  // to a different environment. It's meant for internal testing only.
+  // to a different environment. It's meant for internal testing only and
+  // will be removed in future versions. Use settings in EnhancedBigtableStubSettings
+  // to override the endpoint.
   private static final String MONITORING_ENDPOINT =
```

Comment:
This is a bit confusing now - its no longer the monitoring endpoint, but a monitoring endpoint override from sys prop. How about `MONITORING_ENDPOINT_OVERRIDE_SYS_PROP`?

Reply:
Also mark it as Nullable and deprecated